revolv/static/project.scss

```diff
@@ -43,6 +43,439 @@
     }
 }
 
+body {
+    font-family: Lato;
+}
+
+.row {
+    margin-right: 0px;
+    margin-left: 0px;
+    max-width: 100rem;
+
+    &.heading {
+        height: 500px;
+    }
+
+    &.about, &.project-updates, &.dashboard {
+        margin-right: 15%;
+        margin-left: 15%;
+        max-width: 70%;
+    }
+
+    &.dashboard {
+        height: 600px;
+    }
+
+    &.categories {
+        max-width: 100%;
+    }
+}
+
+.header-img-div {
```

Comment:
having a class of `header-img-div` is kind of weird since technically the element might not be a div at all. How about `header-img-container` instead?